terraform/contentful-backup-bucket.tf

```diff
@@ -0,0 +1,62 @@
+  ### Maybe this should reside with the serverless.yml for the lambda function? 
+  ### https://github.com/UoA-eResearch/research-hub-contentful-backup/blob/master/serverless.yml
```

Comment:
Might be good to make a decision on this one? The options seem to be keeping the terraform file here, putting the equivalent in the serverless.yml, or perhaps folding the contentful backup repo into the monorepo... :thinking: Your thoughts, Rose?

Reply:
I think best to migrate the bucket setup to the serverless.yml. That keeps it nicely packaged as a standalone service. 
But we could also then move it into the monorepo - that part is probably a question for the team.